.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] ..."
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
What should be an example of "Go to"?

Reply:
Go to Grafana Dashboard? Honestly the Bug and Feature request templates are from GitHub. I just deleted the things which were obvious as inapplicable.  So if you have better idea what to put there we should do it.